docs/products/products_general_info.md

```diff
@@ -0,0 +1,3 @@
+# Products guides overview
```

Comment:
What is need for such a short document?

Reply:
All the "overviews" came from the initial implementation of the html. Going to remove, thanks @Boomatang